app/services/storage.js

```diff
@@ -39,6 +39,14 @@ interface ProfileSchema {
   createOrUpdate(storage: ProfileStorage): Promise<number>;
 }
 
+export const SessionStartReason = {
```

Comment:
I tried to find a better way to do enums, and couldn't. Did you come to a good conclusion here? Is this all moot with TypeScript?